src/providers/index.js

```diff
@@ -65,7 +65,6 @@ const performScorecardAnalysis = async (repo) => {
   const start = new Date().getTime()
   const { stdout, stderr } = await exec(`docker run -e GITHUB_AUTH_TOKEN=${process.env.GITHUB_TOKEN} --rm ${ossfScorecardSettings.dockerImage} --repo=${repo.html_url} --show-details --format=json`)
   if (stderr) {
-    console.error(stderr)
```

Comment:
It is not possible to parse a string, so the best option would be to remove this to prevent a token leak.

Or maybe I haven't found the option yet.

Reply:
I know there is `redact` to do that, but it only works at the object level, or that's the only way it has worked correctly for me.

https://github.com/pinojs/pino/blob/main/docs/redaction.md

Reply:
You are totally right! Let me see if I can use a hook for this, as the token pattern is clear.